### PR TITLE
E_CLASSROOM-258 [Admin] [FE/BE] Add/Edit categories functionality

### PR DESCRIPTION
**Pull request by ErickDoroy**
### Issue Link
- https://framgiaph.backlog.com/view/E_CLASSROOM-258

### Definition of Done
- [x] Adding category name and description
- [x] Editing category name and description

### Related PR:
- https://github.com/framgia/sph-classroom-els-fe/pull/115
#### Main note
to add a category
- http://localhost:82/api/v1/admin/add-category
- Change HTTP method to POST
   name:
   description:

to update a category
-  http://localhost:82/api/v1/categories
- Change HTTP method to PATCH
- use x-www-form-urlencoded when inputting data 
   name:
   description:

#### Pages Affected
- 

#### Scenarios/ Test Cases
- N/A

### Screenshots
- N/A

### CODE REVIEW

File: routes/api.php [hunk 1]
```diff
@@ -68,6 +68,7 @@
         Route::get('/quiz_attempts/{quiz_id}', [UserScoresAndAttemptsController::class, 'UserScoreAndAttempts']);
         Route::post('/profileEdit', [ChangeNameEmailController::class, 'changeName']);
         Route::post('/profileEdituploadImage', [UploadImageController::class, 'uploadAvatar']);
+        Route::post('/admin/add-category', [CategoryController::class, 'store']);
```
Comment (karlolee50) [inline routes/api.php:71]: is there no API for editing a category?

Comment (karlolee50) [inline routes/api.php:71]: is there no API for editing a category?

**Comment by catampongan** [comment]
## Approved PR #54 
- Able to add/edit category
- Error is displayed when adding/editing category with the same category name

----------------
Screenshot 2022-02-22 105825

Screenshot 2022-02-22 105935

-------------
- Total Test Time: 0.25 hr
- Total Iterations: 0
- Total Comments: 0

**Review by karlolee50 — APPROVED** [review]
Review time: 10 minutes
Comment/s: 0

**Pull request merged**